contrib/opencensus-ext-azure/examples/logs/properties.py

```diff
@@ -21,4 +21,14 @@
 # and place it in the APPLICATIONINSIGHTS_CONNECTION_STRING
 # environment variable.
 logger.addHandler(AzureLogHandler())
-logger.warning('action', {'key-1': 'value-1', 'key-2': 'value2'})
+
+properties = {'customDimensions': {'key-1': 'value-1', 'key-2': 'value2'}}
```

Comment:
I would follow the [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) standard of naming variables using lower case and underscores.

Reply:
Thanks! I will update!